src/v3/src/transformer/terminal/odaEnrollment/transformOdaEnrollmentLoopback.ts

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { CHALLENGE_METHOD } from '../../../constants';
+import {
+  ButtonElement,
+  ButtonType,
+  DescriptionElement,
+  IdxStepTransformer,
+  ListElement,
+  TitleElement,
+  UISchemaLayout,
+  UISchemaLayoutType,
+} from '../../../types';
+import { copyToClipboard, loc } from '../../../util';
+
+export const transformOdaEnrollmentLoopback: IdxStepTransformer = ({ formBag, transaction }) => {
+  // @ts-expect-error Property 'deviceEnrollment' does not exist on type 'IdxContext' ts(2339)
+  const deviceEnrollment = transaction.context?.deviceEnrollment?.value;
+
+  const {
+    challengeMethod,
+    signInUrl,
+    platform: rawPlatform = '',
+  } = deviceEnrollment;
+  const platform: string = rawPlatform.toLowerCase();
+  const isIOS = platform === 'ios';
+  const isAndroid = platform === 'android';
+
+  formBag.uischema.elements.push({
+    type: 'Title',
+    options: { content: loc('enroll.title.oda', 'login') },
+  } as TitleElement);
+
+  formBag.uischema.elements.push({
+    type: 'Description',
+    options: { content: loc('enroll.explanation.p1', 'login') },
```

Comment:
Is this considered a subtitle? If so, could you add the `contentType: 'subtitle'` prop